indico/modules/events/client/js/reviewing/components/ReviewForm.jsx

```diff
@@ -71,41 +62,40 @@ export default function ReviewForm() {
               <div className="f-self-stretch">
                 <FinalForm
                   onSubmit={() => {}}
-                  initialValues={{comment: '', visibility: 'editors'}}
-                  subscription={{submitting: true}}
+                  initialValues={{comment: '', protected: false}}
+                  subscription={{submitting: true, values: true}}
```

Comment:
you don't need a values subscription here

Reply:
Ah, yeah. Forgot to remove it. At some point I needed that when I wanted to change icon's color based on the selected value :P

Reply:
Tip, you can wrap code that needs the value of a particular field like this:

```jsx
<Field name="thefieldyoucareabout" subscription={{value: true}}>
  {({input: {value}}) => (
    ...
  )}
</Field>
```

That way you only re-render the part that needs it and only when the value of the field you care about changes - avoiding re-renders whenever someone types in a text field.

I used that for `_map_url_template_choice` in RB for example.

Reply:
Yeah, that is pretty useful if you actually want to render something "conditionally". In my case, I wanted to pass a different value as a prop to a component so I think it would be still better to just get it directly from `fprops.values`